setup.py

```diff
@@ -9,7 +9,7 @@
     author='tspurway',
     author_email='tspurway@mozilla.com',
     description='Inferno rules for Tiles Project',
-    install_requires=['psycopg2', 'inferno', 'boto', 'geoip2', 'ua_parser', 'statsd', 'disco'],
+    install_requires=['psycopg2', 'inferno', 'boto', 'geoip2', 'ua_parser', 'statsd', 'disco', 'datadog'],
```

Comment:
the setup script attempts to install disco form pypi which resolves to this project: https://pypi.python.org/pypi/Disco

That's probably not what you mean to install